docs/spec-files/appendix.md

```diff
@@ -0,0 +1,671 @@
+## Appendix
+This section provides test vectors of DIDComm Messages. The test vectors defined in the [Appendix C](#appendix-c-test-vectors) MIGHT be used to validate implementations of [DIDComm Messaging](.). The test vectors  MUST use the sender and recipient private keys defined in the [Appendix A](#appendix-a-secrets-for-test-vectors) to encrypt and sign DIDComm Messages. The test vectors MUST use the sender and recipient public keys defined in the [Appendix B](#appendix-b-diddocs-for-test-vectors) to decrypt DIDComm Encrypted Messages and verify signature of DIDComm Signed Messages.
+
+### Appendix A. Secrets for Test Vectors 
+
+This section provides the sender and recipient private keys. These keys MUST be used for encryption and signing DIDComm Messages.
+
+#### A.1. Sender Secrets
+
+This section defines the sender private keys. 
+
+```json
+[
+   {
+      "kid":"did:example:alice#key-1",
+      "type":"JsonWebKey2020",
+      "verificationMaterial":{
+         "format":"JWK",
+         "value":{
+            "kty":"OKP",
+            "d":"pFRUKkyzx4kHdJtFSnlPA9WzqkDT1HWV0xZ5OYZd2SY",
+            "crv":"Ed25519",
+            "x":"G-boxFB6vOZBu-wXkm-9Lh79I8nf9Z50cILaOgKKGww"
+         }
+      }
+   },
+   {
+      "kid":"did:example:alice#key-2",
+      "type":"JsonWebKey2020",
+      "verificationMaterial":{
```

Comment:
Im not sure what the type of this is... but i sorta wish it , looked like the keys here:

```
  {
    "id": "did:key:z6MkptB2HAniuXsVANcR938iRdkq18GXE4c7uaJPjdp9QFY7#z6MkptB2HAniuXsVANcR938iRdkq18GXE4c7uaJPjdp9QFY7",
    "type": "JsonWebKey2020",
    "controller": "did:key:z6MkptB2HAniuXsVANcR938iRdkq18GXE4c7uaJPjdp9QFY7",
    "publicKeyJwk": {
      "kty": "OKP",
      "crv": "Ed25519",
      "x": "mva13N9iYbjwomoGaEtMzNvmOpoNPj0uuSojznBDh3w"
    },
    "privateKeyJwk": {
      "kty": "OKP",
      "crv": "Ed25519",
      "x": "mva13N9iYbjwomoGaEtMzNvmOpoNPj0uuSojznBDh3w",
      "d": "uxxS11jjJ5JG4V2uH1cz-RqMpJoAy66c2w3M_vuCZl0"
    }
  }
 ```
 
 https://did.key.transmute.industries/generate/ed25519?seed=bb1c52d758e3279246e15dae1f5733f91a8ca49a00cbae9cdb0dccfefb82665d
 
 If it's possible, we should avoid making new data structures.